src/common/api.ts

```diff
@@ -80,6 +80,7 @@ interface CustomAxiosRequestConfig extends Omit<AxiosRequestConfig, 'headers'> {
 export const initializeApi = (
   apiKey: string,
   apiSecret: string,
+  Referer?: string,
```

Comment:
Comme c'est une variable de type string on préferera la faire commencer par une minuscule (comme pour les autres arguments).

```suggestion
  referer?: string,
```

+ mettre cet argument en dernier, après le `apiUrl`